### ISSUE TITLE
Update Standings Page

### ISSUE THREAD

**Issue by prcutler** [prcutler/nflpool#48]
Add HTML to the standings page and TAL content to show standings for each season allowing a player to see the weekly scoring results.

**Comment by prcutler** [comment]
Updated in the dev branch, need to test:  a36b53e845d1b7a383ce599d6305e84ae5d451f5

**Comment by prcutler** [comment]
Undecided if I'm going to do this - I don't know if anyone cares where they stood in a given week once the season is complete

**Comment by prcutler** [comment]
Won't fix
